config/jobs/periodic/docker/periodic-ci-docker.yaml

```diff
@@ -123,6 +123,47 @@ periodics:
             exit $rc
           securityContext:
             privileged: true
+    - name: periodic-integration-cli-docker
+      cluster: k8s-ppc64le-cluster
+      labels:
+              preset-build-docker: "true"
+      cron: 0 14 * * *
+      decorate: true
+      decoration_config:
+        timeout: 3h
+      extra_refs:
+        - base_ref: main
+          org: ppc64le-cloud
+          repo: docker-ce-build
+      reporter_config:
+        slack:
+          channel: 'prow-job-notifications'
+          report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S02N6DWBX0F> <{{.Status.URL}}|View logs>'
+      spec:
+        containers:
+        - image: quay.io/powercloud/docker-ce-build@sha256:f93b6d36ad414d30f584c6555d8c2116de58bd56601da36f4e13d4b5b624114a
+          resources:
+            requests:
+              cpu: "4000m"
```

Comment:
Are you sure this job would also need 4000m cpu like other docker jobs? 
As this one aims at testing CLI, I suppose a lesser resource should suffice. Can you analyse on how much is the cpu required by this job?(Compare resource usage with similar job run upstream on x86)

Reply:
The upstream job uses 4000m but my experiment showed I can safely run `periodic-integration-cli-docker` with 2000 and `periodic-integration-test-docker` with 4000.
https://github.com/ppc64le-cloud/test-infra/compare/master...AshwinHIBM:test-infra:integration-cli#diff-d529e4fce74de0ef2b5f5f6f4131d73c755e5895ad4749261c3a15a2bcc6e61dR147
https://github.com/ppc64le-cloud/test-infra/compare/master...AshwinHIBM:test-infra:integration-cli#diff-d529e4fce74de0ef2b5f5f6f4131d73c755e5895ad4749261c3a15a2bcc6e61dR147